src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -1298,3 +1298,34 @@ def get_user_list(self, zoneName=None):
         multisite_instance = RgwMultisite()
         result = multisite_instance.get_user_list(zoneName)
         return result
+
+
+@APIRouter('/rgw/topic', Scope.RGW)
```

Comment:
We should document all the controllers and APIs
```suggestion
@APIRouter('/rgw/topic', Scope.RGW)
@APIDoc("RGW Topic Management API", "RGW Topic Management")
```